### PR TITLE
testing/py3-fastavro: new aport

### PR DESCRIPTION
**Pull request by gena01**

### CODE REVIEW

File: testing/py3-fastavro/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source="$pkgname-$pkgver.tar.gz::https://github.com/fastavro/fastavro/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	FASTAVRO_USE_CYTHON=1 python3 setup.py build
+}
+
+check() {
+	FASTAVRO_USE_CYTHON=1 python3 setup.py build_ext --inplace
+	PYTHONPATH=${PWD} python3 -m pytest -v tests
```
Comment (maxice8) [inline testing/py3-fastavro/APKBUILD]: can't PYTHONPATH be set to the path of where the build() installed it

Comment (gena01) [reply testing/py3-fastavro/APKBUILD]: This is based on how the repo tests the code itself: https://github.com/fastavro/fastavro/blob/master/run-tests.sh#L21

**Comment by maxice8** [comment]
There are tests that are failing

**Comment by maxice8** [comment]
hmmm, works locally

**Pull request closed without merging**